src/Range.jsx

```diff
@@ -61,8 +61,12 @@ class Range extends React.Component {
     if (nextBounds.length === bounds.length && nextBounds.every((v, i) => v === bounds[i])) return;
 
     this.setState({ bounds: nextBounds });
```

Comment:
Should `nextBounds` have the same value as following `newValues`?

Reply:
If the value is already within the range it shouldn't be changed.